json_examples/task-full_eeg.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
t would be better if this file lives in the same folder as other fMRI/MEG templates:
BIDS-Starter-Kit/templates/sub-01/ses-01/eeg/sub-01_ses-01_task-FullExample_run-01_eeg.json

These templates are meant to be empty.

Is there mean to be a difference with this file that is added here ?
sub-01_task-FullExample_acq-CTF_run-1_proc-sss_eeg.json